src/components/Card/Card.scss

```diff
@@ -1,16 +1,53 @@
+// .card {
+//   display: grid;
+//   grid-template:
+//     "image image image" 1fr
+//     ". title ." auto / var(--space--gap) 1fr var(--space--gap);
+//   border: var(--border);
+//   border-color: var(--color--accent);
+
+//   &__title {
+//     grid-area: title;
+//   }
+//   &__image {
+//     grid-area: image;
+//     mix-blend-mode: overlay;
+//   }
+// }
+
+
+/* Card.css */
 .card {
-  display: grid;
-  grid-template:
-    "image image image" 1fr
-    ". title ." auto / var(--space--gap) 1fr var(--space--gap);
-  border: var(--border);
-  border-color: var(--color--accent);
+  width: 300px;
+  padding: 20px;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  background-color: #fff;
+}
```

Comment:
This card looks so professional, with the subtle styles and nicely rounded corners :)